torch/distributed/rpc.py

```diff
@@ -94,13 +97,16 @@ def rpc_async(to, func, args=None, kwargs=None):
         of ``func`` on ``args`` and ``kwargs`` can be retrieved from the Future
         object.
     """
+    assert _agent is not None, "init_rpc(..) has not been called to setup rpc_agent yet."
     qualified_name = torch.jit._find_builtin(func)
-    if qualified_name is None:
-        raise RuntimeError("unknown builtin function %s." % func)
 
     args = args if args else ()
     kwargs = kwargs if kwargs else {}
-    return invoke_rpc(_agent, to, qualified_name, *args, **kwargs)
+
+    if qualified_name is not None:
```

Comment:
What if our `func` name for python collides with a builtin op name?

Reply:
builitin op start with torch., it should not collide with user defined function?